pkg/graveler/committed/manager_test.go

```diff
@@ -35,16 +36,16 @@ func TestManager_WriteRange(t *testing.T) {
 	}{
 		{
 			name:           "iterator_exhausted",
-			initStorageID:  "",
-			writeStorageID: "",
+			initStorageID:  config.SingleBlockstoreID,
```

Comment:
I don't understand what we did in this test, but also what we did in the other test files.  I would have _expected_ to have twice as many test cases: once for "" aka SIngleBlockstoreID, one for "sid1".  (This could have happened with nested `test.Run()`s, of course.)  But instead some test cases switch to testing "", others to "sid1".  And I don't understand why.

Obviously I don't understand this for all changes in this file, not only for this one.

Reply:
As far as I can understand, we're trying to make sure `StorageID` flows properly from start to finish. By starting with either "" or "sid*" on one side, and making sure we're getting the same result in the other end, we have some confidence it's being passed properly, no?

Reply:
I guess? To be convinced, I need to dig through the code and implementations and see that all interesting places are covered. 
I would have preferred to test everything twice, once with no storage id and once with. Then I wouldn't need to think. 
Not blocking.

Reply:
Makes sense.
Added.